src/utilities/did/useIsOnChainDidDeleted.ts

```diff
@@ -1,18 +1,19 @@
-import { Did, DidUri } from '@kiltprotocol/sdk-js';
+import type { Did } from '@kiltprotocol/types';
+
+import { DidResolver } from '@kiltprotocol/sdk-js';
 
 import { useAsyncValue } from '../useAsyncValue/useAsyncValue';
 
-async function getIsOnChainDidDeleted(
-  did: DidUri | undefined,
-): Promise<boolean> {
+async function getIsOnChainDidDeleted(did: Did | undefined): Promise<boolean> {
   if (!did) {
     return false;
```

Comment:
I know it was here before, but basically if the input DID is nullish, it is considered **not** deleted? As I read the logic below, this function returns `true` if the DID is valid, so the naming of the function is misleading since it gives away the opposite, as far as I can see. Or otherwise, the conditions below are a hard to follow for someone like me 😄 Maybe the boolean condition can be split up into two.

Reply:
I agree it's confusing, but I tried not to change any of Sporran's logic unless the new SDK made it necessary to change it.